index.src.html

```diff
@@ -2768,6 +2829,8 @@
   readonly attribute unsigned long displayHeight;
   readonly attribute unsigned long long? duration;  // microseconds
   readonly attribute long long? timestamp;          // microseconds
+  readonly attribute VideoColorSpace? colorSpace;
```

Comment:
Review note: We could make this non-nullable and use a `VideoColorSpace` instance with null values instead.